src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -416,7 +423,11 @@ public interface IStartConfiguration
             TaskProcessMode ProcessMode { get; set; }
         }
 
-      
+        public interface IStartConfiguration2 : IStartConfiguration
```

Comment:
I'm not sure it was the right choice on my part, but for the interfaces where we have had to do this I've taken the approach of creating a new interface that describes just the new functionality, and making the concrete class implement the original interface and the new one, adding a TODO to merge them back together later, and updating the API changes doc.

will that approach work for you here?